backend/src/main/resources/application.properties

```diff
@@ -49,6 +49,8 @@ internal.admin-role=crc-notifications-team
 
 # Duration rbac entries are kept in cache
 quarkus.cache.caffeine.rbac-cache.expire-after-write=PT120s
+quarkus.cache.caffeine.rbac-cache.metrics-enabled=true
+quarkus.log.category."io.quarkus.cache.runtime.caffeine.CaffeineCacheManagerBuilder".level=DEBUG
```

Comment:
Just to be sure than cache setup values are used.
Could be removed later.

Reply:
Do you know if this could spam the logs with tons of entries?

Reply:
I tested it localy, it just add 2 lines at startup like:
`2024-02-06 14:26:44,651 DEBUG [io.qua.cac.run.caf.CaffeineCacheManagerBuilder] (Quarkus Main Thread) Building Caffeine cache [maintenance] with [initialCapacity=null], [maximumSize=null], [expireAfterWrite=PT1M], [expireAfterAccess=null] and [metricsEnabled=false]
2024-02-06 14:26:44,654 DEBUG [io.qua.cac.run.caf.CaffeineCacheManagerBuilder] (Quarkus Main Thread) Building Caffeine cache [rbac-cache] with [initialCapacity=null], [maximumSize=null], [expireAfterWrite=PT2M], [expireAfterAccess=null] and [metricsEnabled=true]
`

Reply:
My bad, I misread the conf key. This one indeed cannot spam the logs 👍